rosplan_planning_system/src/ProblemGeneration/ProblemInterface.cpp

```diff
@@ -101,7 +101,8 @@ namespace KCL_rosplan {
 			std::ifstream problemIn(req.problem_path.c_str());
 			if(problemIn) res.problem_string = std::string(std::istreambuf_iterator<char>(problemIn), std::istreambuf_iterator<char>());
 		}
-		return success;
+		res.problem_generated = success;
+		return true;
```

Comment:
this would need to go in a separate PR, is not part of the tests but is nice that as part of this PR many bugs are comming out!